nautobot/core/api/serializers.py

```diff
@@ -220,10 +225,11 @@ def build_property_field(self, field_name, model_class):
         """
         Create a property field for model methods and properties.
         """
-        if field_name == "tags":
+        if isinstance(getattr(model_class, field_name, None), (_NautobotTaggableManager, _TaggableManager)):
```

Comment:
Just a note that we'll have to revisit this after #3641, though I don't believe that one is in `develop-to-next-20230421` branch at this time.